jstl/1.2/_index.md

```diff
@@ -0,0 +1,55 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+date: 2019-07-18T16:10:38-04:00 
```

Comment:
The time component can be removed.

Reply:
You removed both the date and the time.  We still need the date: header.
date: 2019-07-18